### PR TITLE
Empty State for Assessments Page

### LINKED ISSUE #10965: Add empty state for assessments page
The assessments page within a course instance should have a meaningful empty state that briefly describes what an assessment is, links to the documentation for more information, and provides a clear call to action to create a new assessment.

The page is implemented in [`apps/prairielearn/src/pages/instructorAssessments`](https://github.com/PrairieLearn/PrairieLearn/tree/master/apps/prairielearn/src/pages/instructorAssessments).

### PR DESCRIPTION
**Pull request by miguelaenlle**
Closes #10965

On the `instructorAssessments` page, if a course instance has no assessments, the following empty state is displayed:

image

### CODE REVIEW

File: apps/prairielearn/src/pages/instructorAssessments/instructorAssessments.html.ts
Comment (nwalters512) [inline apps/prairielearn/src/pages/instructorAssessments/instructorAssessments.html.ts:153]: cc @mfsilva22 for review of copy.

**Review by nwalters512 — APPROVED** [review]
Looks great! Let's wait for Mariana to review the copy before we merge this.

File: apps/prairielearn/src/pages/instructorAssessments/instructorAssessments.html.ts
Comment (mfsilva22) [reply apps/prairielearn/src/pages/instructorAssessments/instructorAssessments.html.ts:153]: looks great!

**Review by nwalters512 — APPROVED** [review]

**Pull request merged**
